lib/active_admin/resource_dsl.rb

```diff
@@ -65,6 +65,10 @@ def permit_params(*args, &block)
       belongs_to_param = config.belongs_to_param
       create_another_param = :create_another if config.create_another
 
+      if config.build_form_with_permitted_params_only
+        config.permitted_params = block ? block.call : args.flatten
```

Comment:
This block needs to be evaluated in the context of a controller, we can have `params` or `current_user` calls here. So I don't think this is going to work.

Mayb you can skip this and instead call `controller.permitted_params` when rendering the form?

Reply:
I implemented this suggestion in https://github.com/activeadmin/activeadmin/pull/5722/commits/e5f16cb8dc02a786c196a2b58c3918703db373f8.